app.py

```diff
@@ -655,6 +655,58 @@ def route_home():
         )
 
 
+@app.route("/how-to")
+def route_how_to():
+    try:
+        # today = datetime.date.today().isoformat()
+        current = get_current_audit()
```

Comment:
Is the audit data used in the (presumably) text-only template? Maybe replace with `{}`, or tweak the templates?